src/axom/core/memory_management.hpp

```diff
@@ -21,11 +21,8 @@
 
 namespace axom
 {
-#ifdef AXOM_USE_UMPIRE
-const int DEFAULT_ALLOCATOR_ID =
-  umpire::ResourceManager::getInstance().getAllocator("HOST").getId();
```

Comment:
🎉

Reply:
Thanks again for your help @davidbeckingsale